client/src/charts/NivoCharts.js

```diff
@@ -1,13 +1,19 @@
 import { ResponsiveBar } from '@nivo/bar';
 import { ResponsiveLine } from '@nivo/line';
 import { ResponsivePie } from '@nivo/pie';
+import { ResponsiveBubble } from '@nivo/circle-packing';
 import { formats, dollar_formats } from "../core/format.js";
 import { Fragment } from 'react';
 import { IconZoomIn, IconZoomOut } from '../icons/icons.js';
+import { create_text_maker_component } from '../components/index.js';
 import { trivial_text_maker } from '../models/text.js';
 import { breakpoints } from '../core/breakpoint_defs.js';
+import { newIBCategoryColors } from '../core/color_schemes.js';
 import MediaQuery from 'react-responsive';
 import './NivoCharts.scss';
+import graph_text from './NivoCharts.yaml';
```

Comment:
Not a totally strict code-style rule, but I try to keep imports from same-named yaml and scss modules as the the very first line of same-named js file (usually followed by a lone break and all npm modules then a line break and local modules, unless some other grouping of imports is more meaningful in the file).

Reply:
I still plan on breaking up NivoCharts, so it would be a generic text to go along with those multiple js files.